Formula/platformio.rb

```diff
@@ -13,16 +13,16 @@ class Platformio < Formula
     sha256 "15d1add995b55416e0d42dad9e884e099416a14650e975e90ec91ff6a418a5bf" => :sierra
   end
 
-  depends_on "python@2" # does not support Python 3
+  depends_on "python" # Homebrew's Python 2.7.x (if installed) otherwise the macOS system Python
```

Comment:
This comment is inaccurate. `python` formula is Homebrew's Python 3. Does this build with Python 3 or not?

Reply:
What do you mean? I used your docs https://docs.brew.sh/Homebrew-and-Python

Yes, PlatformIO 4.0 is compatible with Python 2 and Python 3

Reply:
I removed that comment. Please update docs.